instrumentation/jaxws/jaxws-jws-api-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxws/jws/v1_1/JwsAnnotationsInstrumentation.java

```diff
@@ -77,8 +74,6 @@ public static void startSpan(
 
       Context parentContext = currentContext();
       request = new JaxWsRequest(target.getClass(), methodName);
-      ServerSpanNaming.updateServerSpanName(
```

Comment:
This would have been relevant only when request is handled by a jax-ws framework that we don't have integration with (or when framework instrumentation is disabled or when there is a bug that prevents it from working properly) because jax-ws framework instrumentation runs before it and already sets the name.

Reply:
Yep -- and it sets a `<class>.<method>` span name, so I thought we won't lose that much (if anything) if I remove that.